src/main/java/org/cactoos/text/UncheckedText.java

```diff
@@ -33,6 +33,10 @@
  * <p>There is no thread-safety guarantee.
  *
  * @since 0.3
+ * @todo #1135:30min All classes implementing Text need to be refactored
```

Comment:
@ccostin93 this class is not concerned by this todo actually, can you select another one to put it instead?

Reply:
@victornoel The only other remaining class if TextEnvelope.

Does this mean that the puzzle is complete?

Reply:
@ccostin93 exactly then, let's remove it

Reply:
@victornoel done